Build error generated mac el capitan

`/Users/zarboz/hyperion/libsrc/utils/Logger.cpp:49:25: error: use of undeclared identifier 'program_invocation_short_name'
        _appname = std::string(program_invocation_short_name);
                               ^
/Users/zarboz/hyperion/libsrc/utils/Logger.cpp:61:12: error: use of undeclared identifier 'program_invocation_short_name'
                openlog (program_invocation_short_name, LOG_CONS | LOG_PID | LOG_NDELAY, LOG_LOCAL0);
                         ^
2 errors generated.
`

--- a/libsrc/utils/Logger.cpp
+++ b/libsrc/utils/Logger.cpp
@@ -46,7 +46,11 @@ Logger::Logger ( std::string name, LogLevel minLevel ):
 	_syslogEnabled(true),
 	_loggerId(loggerId++)
 {
+#ifdef GLIBC
 	_appname = std::string(program_invocation_short_name);
+#else
+	_appname = std::string(getprogname());
+#endif
 	std::transform(_appname.begin(), _appname.end(),_appname.begin(), ::toupper);
 
 	loggerCount++;

